### ISSUE TITLE
alag + linCmt() seems to have some problems

### ISSUE THREAD

**Issue by mattfidler** [nlmixrdevelopment/RxODE#258]
``` r
library(RxODE)
#> RxODE 0.9.2.0 using 4 threads (see ?getRxThreads)
library(ggplot2)
library(dplyr)
#> 
#> Attaching package: 'dplyr'
#> The following objects are masked from 'package:stats':
#> 
#>     filter, lag
#> The following objects are masked from 'package:base':
#> 
#>     intersect, setdiff, setequal, union
library(lotri)

RxODE({
  ka=1;
  cl=3.5;
  vc=40;
  Conc = linCmt()
  alag(depot) = 6
})%>%
et(dose=100, time=0, addl=6, ii=24)%>%
et(0, 250, by=0.1) %>%
rxSolve()%>%
plot(Conc)
```

![](https://i.imgur.com/0wZsNU7.png)

<sup>Created on 2020-08-03 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Related to #240

**Comment by mattfidler** [comment]
``` r
library(RxODE)
#> RxODE 0.9.2.0 using 4 threads (see ?getRxThreads)
library(ggplot2)
library(dplyr)
#> 
#> Attaching package: 'dplyr'
#> The following objects are masked from 'package:stats':
#> 
#>     filter, lag
#> The following objects are masked from 'package:base':
#> 
#>     intersect, setdiff, setequal, union
library(lotri)

RxODE({
  ka=1;
  cl=3.5;
  vc=40;
  Conc = linCmt()
  alag(depot) = 1
})%>%
et(dose=100, time=0, addl=6, ii=24)%>%
et(0, 250, by=0.1) %>%
rxSolve()%>%
plot(Conc)
```

![](https://i.imgur.com/qV20Vbw.png)

<sup>Created on 2020-08-03 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by mattfidler** [comment]
Seems to be fixed with #258 
``` r
library(RxODE)
#> RxODE 0.9.2.0 using 4 threads (see ?getRxThreads)
m258 <- RxODE({
  ka=1;
  cl=3.5;
  vc=40;
  Conc = linCmt()
  alag(depot) = 1
})

m258o <- RxODE({
  ka=1
  cl=3.5
  vc=40
  d/dt(depot) = -ka * depot
  d/dt(central) = ka * depot - cl / vc * central
  Conc = central / vc
  alag(depot) = 1
})

s1 <- m258 %>%
  et(dose=100, time=0, addl=6, ii=24)%>%
  et(0, 250, by=0.1) %>%
  rxSolve()

s2 <- m258o %>%
  et(dose=100, time=0, addl=6, ii=24)%>%
  et(0, 250, by=0.1) %>%
  rxSolve()

plot(s1, Conc)
```

![](https://i.imgur.com/SARXTe2.png)

``` r

plot(s2, Conc)
```

![](https://i.imgur.com/aqW6RI7.png)

<sup>Created on 2020-08-13 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>
